### DISCUSSION TITLE
How Repair works and should I bother?

### DISCUSSION THREAD

**Discussion by Godd67** [immich-app/immich#7631] [Q&A]
On the Administration/Repair screen I have >7k untracked files .  
Should I do anything about it? Do I need them? Do they take space? Can I delete them?
There is a button RepairAll, but it is disabled. Is there a trick to enable it?

Thanks!

**Comment by mmomjian** [comment]
I'm not sure about the repair button, but 7k untracked files is not normal / good. How many photos total are in your library? How did you import them? Do you use the mobile app?

Can you view the untracked files with a file browser? Are they actual images?

**Reply by Godd67** [reply]
I actually have >75k of untracked files. All of them reside in /usr/src/app/upload/thumbs and encoded-video folders so I assume they are lost thumbnails or orphan transcoding incidents. 
I have 2 external libraries that shared among 2 users - total amount probably about 75k even though I did not count.
I found similar topic - https://github.com/immich-app/immich/discussions/7631 , looks like people say do not bother with it ....
My only question is do these files eat space or affect performance in any way?
